optimum/habana/transformers/models/modeling_all_models.py

```diff
@@ -113,6 +114,46 @@ def gaudi_conv1d_forward(self, x):
     return x
 
 
+# Adapted from transformers.modeling_utils.PreTrainedModel._check_and_enable_sdpa
+@classmethod
+def gaudi_check_and_enable_sdpa(cls, config, hard_check_only: bool = False) -> PretrainedConfig:
+
+    #This model doesn't support SDPA in Gaudi yet, fallback to original code.
+    MODELS_ATTN_IMPLEMENTATION_EAGER = [
+        "gpt_bigcode",
+        "mistral",
```

Comment:
@vivekgoe  should we add the t5 model here too?

Reply:
you mean "bart". yes please go ahead and add it.